deployment/network-operator/templates/tests/test-dev-plugin.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.deployCR .Values.devicePlugin.deploy  }}
+{{- if and .Values.deployCR .Values.rdmaSharedDevicePlugin.deploy  }}
```

Comment:
a comment about the name of the file, should we change to `test-rdma-shared-dev-plugin.yaml` ?
im fine either way, but thought its worth mentioning

Reply:
Good catch, Adrian! I'll rename it once CI reports on it